dataset.py

```diff
@@ -32,6 +32,27 @@ def gen_tabular_dataset(monthly: bool, samples_per_site: int) -> pd.DataFrame:
                                (-55,-3.5,),(-54,-1,),(-52.5,-13,),(-51.5,-2.5,)],
                                          0.5)
 
+def add_features_from_rasters(df: pd.DataFrame, rasters: list) -> pd.DataFrame:
+  '''
+  Given a pd.DataFrame df:
+  1. enumerates each row looking for a 'lat' and 'lon' column
+  2.   for each raster, looks up the lat and lon value
+  3.     adds that value to the feature_df
+  4. returns the concat of both original df and new feature_df
+  '''
+  feature_dict = {}
+  for raster in rasters:
+    feature_dict[raster.name] = []
+  
+  for row in df.itertuples():
+    lat = getattr(row, "lat")
```

Comment:
Why are we using `getattr(row, "lat")` as opposed to `row["lat"]`?

Reply:
fixed

Reply:
actually, because its a tuple, you use getattr.  or at least row["x"] doesnt work for me